client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -311,14 +339,18 @@ const PluginsBrowser = React.createClass( {
 		}
 
 		const navigation = this.props.category ? this.getNavigationBar() : this.getSearchBar();
-		const manageButton = this.shouldShowManageButton() && this.getManageButton();
 
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
There might be a way to organise things so that the header buttons become a separate component, with the actual buttons passed in as children, or maybe props to control which buttons to show. That would take care of the lint warning and a little bit of duplication. Would be for a different PR, though.